### PR TITLE
[12] Style the Contact Us Page using CSS

### PR DESCRIPTION
**Pull request by Egwolo2**
## Task: ` To style the contact us page using CSS`
- `Created and styled the  contact us page, added some icons to the image folder. 

Contact laptop
contact ipad1
contact ipad2
contact phone1

`
contact phone2
contact phone3

### CODE REVIEW

File: submissions/egwoloOkorodudu/css/contact.css [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+    margin: 0;
+    font-family: 'Open Sans', sans-seri;
+    line-height: 1.5;
+    height: 100%;
+}
+/*---header section----*/
+.header{
+    overflow: hidden;
+    background-color: #b6249e;
+    padding: 25px 4em 0 20px;
```
Comment (ToluAkin) [inline submissions/egwoloOkorodudu/css/contact.css:14]: Use the same unit all through @Egwolo2

**Review by ToluAkin — CHANGES_REQUESTED** [review]

**Review by adaugochi — COMMENTED** [review]
Well done @Egwolo2 
I love your design. Let us know when you fix the comment on your PR

File: submissions/egwoloOkorodudu/css/contact.css (hunk 1 above)
Comment (Egwolo2) [reply submissions/egwoloOkorodudu/css/contact.css:14]: Okat @ToluAkin 
Noted

Comment (Egwolo2) [reply submissions/egwoloOkorodudu/css/contact.css]: Corrections has been made @ToluAkin  and @adaugochi 
Thanks a lot

**Review by ToluAkin — APPROVED** [review]

**Pull request merged**